CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 - Improved `to_pandas` to persist the original timezone offset for TIMESTAMP_TZ type.
 - Improved `dtype` results for TIMESTAMP_TZ type to show correct timezone offset.
 - Improved error message when passing non-bool value to `numeric_only` for groupby aggregations.
+- Removed unnecessary warning about sort algorithm in `sort_values`.
```

Comment:
This seems so minor, is it even worth calling out as an improvement?

Reply:
yeah, it is pretty minor. But it does have user facing impact (right now this warning shows up in notebook in red) so might be good include this here.